src/services/mealAPI.js

```diff
@@ -35,3 +36,14 @@ export async function getMealIngredients() {
 export function mealIngredientImg(ingredientName) {
   return `https://www.themealdb.com/images/ingredients/${ingredientName}.png`;
 }
+
+export async function getMeals() {
+  const doze = 12;
+  try {
+    const response = await fetch(MEALS);
+    const data = await response.json();
+    return data.meals.slice(0, doze);
+  } catch (erro) {
+    console.log('Erro in get all meats');
+  }
+}
```

Comment:
Ficou ótima a implementação. Diria que daria pra deixar melhor ainda fazendo com que a função pudesse receber como parâmetro um número variável de dados esperados. Segue a sugestão de modificação. Nesse caso coloquei o `dataQty = -1` para dizer que padrão é a função `getMeals` vai retornar todos os dados que vieram da resposta da api.
```suggestion
export async function getMeals(dataQty = -1) {
  try {
    const response = await fetch(MEALS);
    const data = await response.json();
    return data.meals.slice(0, dataQty);
  } catch (erro) {
    console.log('Erro in get all meats');
  }
}
```